README.md

```diff
@@ -213,13 +213,15 @@ class EmailEvent < ActiveRecord::Base
     opened    3, processor_class: EmailOpenedProcessor
     delivered 4, processor_class: DeliveryProcessor
   end
-
-  def process
-    class.event_types[event_type][:processor_class].new(self).process
-  end
 end
 ```
 
+Custom configuration parameters are available as an instance method on the object as well.
+```
+e = EmailEvent.new(event_type: 1)
+e.event_type_options # => { processor_class: RejectedProcessor, value: 1 }
```

Comment:
We might consider renaming this to `event_type_details` or something else without the word "options". The word "option" is used elsewhere to mean an available enum value. Here, "options" means the configuration attributes of the current "option".